examples/tic-tac-toe/server/server.bal

```diff
@@ -88,10 +88,12 @@ service class GameServer {
             started = true;
         }
 
+        next = sign == SIGN_X ? SIGN_O: SIGN_X;
+
         UserMove msg = {
             'type: "move",
             move: squareNumber,
-            next: sign == SIGN_X ? SIGN_O: SIGN_X,
+            next: next,
             player: sign
```

Comment:
```ballerina
UserMove msg = {
    'type: "move",
    move: squareNumber,
    next: sign == SIGN_X ? SIGN_O: SIGN_X,
    next,
    player: sign
};
```
That should work.

Reply:
We need to update the global `next` variable to keep the status. If the user refreshes the page it keeps track of the next allowed player(whether `X` or `O`).

Reply:
Sorry for the confusion. Updated the code with [2ea4595](https://github.com/ballerina-platform/module-ballerina-websocket/pull/667/commits/2ea4595dc091289d231eba0b47b3521014d73834)